doc/user-guide/hierarchical-data.rst

```diff
@@ -1,5 +1,3 @@
-.. currentmodule:: datatree
-
 .. _hierarchical-data:
 
 Working With Hierarchical Data
```

Comment:
```suggestion
Hierarchical data
```
This header shows up in the table of contents, and "Working with" is a little redundant